nsxt/provider.go

```diff
@@ -1090,6 +1090,12 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		return nil, err
 	}
 
+	initNSXVersion(getPolicyConnector(clients))
+	if util.NsxVersionHigherOrEqual("9.0.0") {
+		// Disable MP client from 9.0.0 onwards
```

Comment:
This is fine for now, but ideally we should not invoke `configureNsxtClient` when NSX is 9.0.0 and above.
This would require us to rewrite the session create API which is currently handle fired from inside `configureNsxtClient`. We should do this regardless, because it paves the way of removing the old SDK when the time comes.

Reply:
I've removed this code. I will handle the change in session create API with a new commit in this PR or a new PR